javascripts/govuk/selection-buttons.js

```diff
@@ -1,6 +1,11 @@
+// DEPRECATED
+// This isn’t needed if you’re using GOV.UK Elements 3.0.0 or above
+
 ;(function (global) {
   'use strict'
 
+  console.warn('Deprecation warning: Custom radio buttons and checkboxes (released in GOV.UK Elements 3.0.0) no longer require this JavaScript.')
```

Comment:
We'd have to do the full whammy `if (window.console && window.console.warn) {}` like https://github.com/alphagov/govuk_prototype_kit/blob/6a38393eb657394fd21f7cbad2efb8ea8d709840/app/assets/javascripts/application.js#L7